src/systems/wheel_slip/WheelSlip.cc

```diff
@@ -242,9 +245,30 @@ bool WheelSlipPrivate::Load(const EntityComponentManager &_ecm,
 /////////////////////////////////////////////////
 void WheelSlipPrivate::Update(EntityComponentManager &_ecm)
 {
-  for (const auto &linkSurface : this->mapLinkSurfaceParams)
+  for (auto &linkSurface : this->mapLinkSurfaceParams)
   {
-    const auto &params = linkSurface.second;
+    auto &params = linkSurface.second;
+    const auto * wheelSlipCmdComp =
+      _ecm.Component<components::WheelSlipCmd>(linkSurface.first);
+    if (wheelSlipCmdComp) {
```

Comment:
Style nit: I don't know about the Ignition style-guide, but it seems like the rest of the code in this file puts open-braces on a new line
```suggestion
    if (wheelSlipCmdComp)
    {
```

Reply:
Fixed in https://github.com/ignitionrobotics/ign-gazebo/pull/1241/commits/25a11e63ce5b3e1486f62996b937f6bdf868c56a.